x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -186,27 +225,45 @@ export class ESSearchSource extends AbstractESSource {
   }
 
   // Returns sort content for an Elasticsearch search body
-  _buildEsSort() {
+  _buildEsSort(): Array<Record<string, SortDirection | SortDirectionNumeric>> {
```

Comment:
You can remove `SortDirectionNumeric` since only `SortDirection` is ever returned